### PR TITLE
New  documentation and tutorials

### PR DESCRIPTION
**Pull request by raquellrios**
## Description
KinoML new tutorials, docs and figures.

## Todos
Final docs and tutorials for KinoML tech paper

## Status
- [x] Ready to go

### CODE REVIEW

File: docs/kinoml/data/README.md
Comment (ijpulidos) [inline docs/kinoml/data/README.md:1]: Do we need to add all the files in the `kinoml/data` subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/electron_densities/4f8o_phases.mtz
Comment (ijpulidos) [inline docs/kinoml/data/electron_densities/4f8o_phases.mtz:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/electron_densities/__init__.py
Comment (ijpulidos) [inline docs/kinoml/data/electron_densities/__init__.py:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/kinomescan/DiscoverX_489_Kinase_Assay_Construct_Information.csv
Comment (ijpulidos) [inline docs/kinoml/data/kinomescan/DiscoverX_489_Kinase_Assay_Construct_Information.csv:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/kinomescan/journal.pone.0181585.s004.csv
Comment (ijpulidos) [inline docs/kinoml/data/kinomescan/journal.pone.0181585.s004.csv:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/look_and_say.dat
Comment (ijpulidos) [inline docs/kinoml/data/look_and_say.dat:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/molecules/__init__.py
Comment (ijpulidos) [inline docs/kinoml/data/molecules/__init__.py:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/molecules/chloroform.pdb
Comment (ijpulidos) [inline docs/kinoml/data/molecules/chloroform.pdb:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/molecules/chloroform.sdf
Comment (ijpulidos) [inline docs/kinoml/data/molecules/chloroform.sdf:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/molecules/chloroform_acetamide.pdb
Comment (ijpulidos) [inline docs/kinoml/data/molecules/chloroform_acetamide.pdb:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/molecules/chloroform_acetamide.sdf
Comment (ijpulidos) [inline docs/kinoml/data/molecules/chloroform_acetamide.sdf:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/object_model.png
Comment (ijpulidos) [inline docs/kinoml/data/object_model.png:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/proteins/README.md
Comment (ijpulidos) [inline docs/kinoml/data/proteins/README.md:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: docs/kinoml/data/proteins/__init__.py
Comment (ijpulidos) [inline docs/kinoml/data/proteins/__init__.py:1]: Do we need to add all the files in the kinoml/data subdir here as well? We should avoid having two copies of the same files and data here.

File: kinoml/data/fig_1_kinomltechpaper_v2.png
Comment (ijpulidos) [inline kinoml/data/fig_1_kinomltechpaper_v2.png:1]: I love the summary in this figure, great work!

**Review by ijpulidos — CHANGES_REQUESTED** [review]
This is looking great! Amazing work. I just added a few comments on some changes I think we should make to the install instructions.

Also, do we need to have a copy of the `kinoml/data` subdir in the docs directory? Seems like we should avoid having copies of the data here, as possible.

**Review by AndreaVolkamer — APPROVED** [review]
Looks great. thanks!

**Comment by AndreaVolkamer** [comment]
I think the changes requested by @ijpulidos were addressed by @raquellrios, so I will go ahead and merge the PR.

**Pull request merged**